runtime/test_framework_environment.go

```diff
@@ -0,0 +1,43 @@
+/*
+ * Cadence - The resource-oriented smart contract programming language
+ *
+ * Copyright 2019-2022 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package runtime
+
+import (
+	"github.com/onflow/cadence/runtime/interpreter"
+	"github.com/onflow/cadence/runtime/sema"
+)
+
+type testFrameworkEnvironment struct {
+	*interpreterEnvironment
+}
+
+func NewTestFrameworkEnvironment(
+	config Config,
+	interpreterConfig *interpreter.Config,
+	configConfig *sema.Config,
+) *testFrameworkEnvironment {
+	env := NewBaseInterpreterEnvironment(config)
+
+	env.CheckerConfig = configConfig
+	env.InterpreterConfig = interpreterConfig
```

Comment:
Is overwriting the configs here intentionally?

Reply:
ah, this is a left-over. Removed the file.